packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -890,6 +911,23 @@ describe('SignatureController', () => {
         ).version,
       ).toBe(SignTypedDataVersion.V3);
     });
+
+    // eslint-disable-next-line jest/no-disabled-tests
+    it.skip('invoke decoding api for permits', async () => {
```

Comment:
Should we have additional unit tests to verify the final decoded response is saved, and also that errors are caught and saved?

Perhaps in a new `Decoding` describe?

Reply:
yep adding those.